LGTV/__init__.py

```diff
@@ -158,9 +159,10 @@ def main():
     else:
         try:
             kwargs = parseargs(args.command, args.args)
-        except Exception as e:
-            parser.print_help()
-            sys.exit(1)
+        except Exception:
+            if args.command not in {"sendButton"}:
```

Comment:
I'm not quite satisfied with this hack to add support for an extra command that isn't on the `LGTVRemote` class.

I wonder if we can do something to make exposing commands to the CLI cleaner, but that might be for future work.